### ISSUE TITLE
Allow more flexible linking of individuals and phenotypes

### ISSUE THREAD

**Issue by david4096** [ga4gh/schemas#610]
via @sarahhunt 
> Attaching disease to BioSample seems odd - many sequenced samples are not associated with a disease; if they were it would be more normal to flag this at the individual level. 
This model does not allow for individuals with multiple diseases. 
Disease is a limiting term - what about other traits which may be investigated?
Wouldn't this be better modelled as a record linking an individual to a phenotype/trait/disease with date of linkage?

**Comment by helenp** [comment]
If I edit in a mutation into a normal iPSC to mimic a disease - then this is the BioSample and not the individual. How else would we model this?

**Comment by sarahhunt** [comment]
Thanks for the example use case @helenp. So we need to be able to hold such information at both levels.

**Comment by mbaudis** [comment]
Yes. But we have to document this better. Also to make it very explicit that the "disease" is inherent in the sample, with examples. E.g. peripheral blood from a patient with CRC does not have a ```disease``` value CRC. This is a cause of endless grief when mapping data;

**Comment by david4096** [comment]
It seems like BioSample is the basis for a preparation. Is it possible we need another entity to better describe these experiments? For example, would it make more sense to model @helenp's use case as a *preparation* of a biosample? For example, I may have a single piece of tissue that will be treated three different ways. Are these three new biosamples, or three preparations of the same BioSample?

**Comment by diekhans** [comment]
The current model collapses layers of more complete models.  A
sample is split into aliquots for analysis which are then used
to produce libraries.  In the current GA4GH metadata model,
these aliquot and library would be in experimenter.  It's still
the same sample.

This model might evolve once we get more experience.

**Comment by david4096** [comment]
Although I'm not suggesting we use TCGA barcodes, I found this image helpful in thinking about the biosample data coverage.

![screen shot 2016-11-21 at 1 55 37 pm](https://cloud.githubusercontent.com/assets/3795586/20502270/3661d224-aff2-11e6-8c42-947db7f43637.png)
